app/code/Magento/CatalogInventory/Test/Unit/Model/Quote/Item/QuantityValidator/QuoteItemQtyListTest.php

```diff
@@ -0,0 +1,68 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\CatalogInventory\Test\Unit\Model\Quote\Item\QuantityValidator;
+
+use Magento\Framework\TestFramework\Unit\Helper\ObjectManager;
+use PHPUnit\Framework\TestCase;
+use Magento\CatalogInventory\Model\Quote\Item\QuantityValidator\QuoteItemQtyList;
+
+/**
+ * Class QuoteItemQtyListTest
+ */
+class QuoteItemQtyListTest extends TestCase
+{
+    /**
+     * @var QuoteItemQtyList
+     */
+    private $quoteItemQtyList;
+
+    /**
+     * @var int
+     */
+    private $itemQtyTestValue;
```

Comment:
Just a minor adjustment. If we don't need to share value between different methods, there's no need to create an additional class property. You may use just local variables inside of the methods.

So `$this->itemQtyTestValue` will be simply `$itemQtyTestValue` inside of every method (test) of the current class

Reply:
@rogyar Thank you, I agree, I had some ideas about using it, but then changed my mind and just used this as this is already created